docs/integrations/app-development/opentelemetry/jira-opentelemetry.md

```diff
@@ -0,0 +1,235 @@
+---
+id: jira-opentelemetry
+title: Jira - OpenTelemetry Collector
+sidebar_label: Jira - OTel Collector
+description: The Sumo Logic App for Jira provides insight into Jira user access, request activity, issues, security, sprint events, and user events.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<img src={useBaseUrl('img/integrations/app-development/jira.png')} alt="Thumbnail icon" width="50"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
+
+The Sumo Logic App for Jira provides insight into Jira usage, request activity, issues, security, sprint events, and user events. 
+
+Jira logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Jira-OpenTelemetry/Jira-Schematics.png' alt="Schematics" />
```

Comment:
@sumoanema The majority of these AWS images (all but Dashboards) are broken. Can you confirm the permissions and publish status for these?

Reply:
@kimsauce - These images are present in the gdoc draft already. What I know as part of the process, doc team is responsible for publishing these images in s3 and make the doc point to the same. Nishant was suppose to do it before he left.